samples/Grafana/README.md

```diff
@@ -0,0 +1,55 @@
+### Was this documentation helpful? [Share feedback](https://www.research.net/r/DGDQWXH?src=documentation%2Flocalmachine)
+
+# Configure `dotnet monitor` in an AKS cluster for Managed Grafana
+
+`dotnet monitor` provides snapshots of .NET metrics in the Prometheus exposition format. [Prometheus](https://prometheus.io/docs/introduction/overview/) in turn collects metrics from targets by scraping metrics HTTP endpoints.
```

Comment:
Do we want to link to our metrics endpoint https://github.com/dotnet/dotnet-monitor/blob/main/documentation/api/metrics.md